python/lsst/meas/base/compensatedGaussian/_compensatedGaussian.py

```diff
@@ -119,44 +105,68 @@ def __init__(
             mask_str = f"{base_key}_mask_bits"
             mask_key = schema.addField(mask_str, type=np.int32, doc="Mask bits set within aperture.")
 
-            self.width_keys[width] = (flux_key, err_key, mask_key)
+            # failure flags
+            failure_flag = flagDefs.add(f"{width}_flag", "Compensated Gaussian measurement failed")
+            oob_flag = flagDefs.add(f"{width}_flag_bounds", "Compensated Gaussian out-of-bounds")
+
```

Comment:
I think `width` evaluates to to a number (integer).  Do you think it will be apparent "enough" that this number in the flag name is the kernel width?  And will the casual user be expected to translate `bounds` to `outOfBounds` internally (probably yes to both, but my ~10pm brain didn't jump there right away)?

Reply:
That's the name of the measurement ... `base_compensatedGaussian_5` or whatever.  It's what we decided on ages ago, and it may not even matter since the compensated gaussians are not working well as a calibration flux measurement.  😞 
As for the `bounds`, this is also the name from before, except done correctly.  I don't think this is going to be used for anything anyway.